client/webserver/site/src/js/markets.ts

```diff
@@ -975,13 +975,13 @@ export default class MarketsPage extends BasePage {
       qtyConv = market.quoteUnitInfo.conventional.conversionFactor
     }
     return {
-      host: market.dex.host,
+      host: market?.dex?.host,
       isLimit: limit,
       sell: sell,
-      base: market.base.id,
-      quote: market.quote.id,
+      base: market?.base.id,
+      quote: market?.quote.id,
```

Comment:
The result is now instead of a javascript error, we return an incorrect result to the caller of `parseOrder`.

Reply:
I think the backend would return an error for an invalid host/data, no?

Reply:
> Maybe but please consider that using the optional chaining parameter is in general not always a good idea. Instead of console errors we now have incorrect behavior.

Noted. Most of the places it is used in this PR are where we do equality comparisons and I think we won't risk incorrect behaviour there.

Reply:
@chappjc, I believe your concerns have been addressed in https://github.com/decred/dcrdex/commit/d0c56ae4e8fd1a3d143a0d73c9206499a0c4d261, right?